images/github-cli/Dockerfile

```diff
@@ -7,7 +7,17 @@ RUN apt-get update -qq && apt-get install -y ca-certificates git wget
 ADD https://cli.github.com/packages/githubcli-archive-keyring.gpg /usr/share/keyrings/githubcli-archive-keyring.gpg
 RUN chmod 644 /usr/share/keyrings/githubcli-archive-keyring.gpg
 RUN echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" > /etc/apt/sources.list.d/github-cli.list
-RUN apt-get update -qq && apt-get install -y gh
+RUN echo "deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_11/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list"
```

Comment:
You're adding the kubic apt repo here but not adding its signing key until after you've installed the podman package. Doesn't this mean you're installing the package from the Debian distro rather than from kubic?

Is there a reason why we need to use the kubic repo rather than vanilla Debian for installing podman?

Reply:
I'll remove the signing key for the kubic repo, it's not suggested for production use anyways.